packages/terra-form-select/src/native-select/NativeSelect.jsx

```diff
@@ -0,0 +1,247 @@
+import React, {
+  useState,
+  useRef,
+} from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import classNamesBind from 'classnames/bind';
+import { injectIntl, intlShape } from 'react-intl';
+import ThemeContext from 'terra-theme-context';
+import {
+  defaultPlaceholderValue,
+  isCurrentPlaceholder,
+  getDisplay,
+  getFirstValue,
+  getOptGroupKey,
+  geOptGroupKeyIndex,
+} from './NativeUtils';
+import {
+  optionPropType,
+  optGroupPropType,
+} from './NativePropTypes';
+
+import styles from './NativeSelect.module.scss';
+
+const cx = classNamesBind.bind(styles);
+
+const propTypes = {
+  /**
+   * The string containing ids for elements to describe the select.
+   */
+  ariaDescribedBy: PropTypes.string,
+  /**
+   * The aria label string value for the select.
+   */
+  ariaLabel: PropTypes.string.isRequired,
+  /**
+   * Additional attributes to spread directly onto the html select node.
+   * This prop is provided as standard customProps are spread onto the outer containing element which is not the html select in this case.
+   */
+  // eslint-disable-next-line react/forbid-prop-types
+  attrs: PropTypes.object,
+  /**
+   * The default value of the select. Can be a string, or number.
+   */
+  defaultValue: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+  /**
+   * Whether the input is disabled.
+   */
+  disabled: PropTypes.bool,
+  /**
+   * The Select identifier. Links the htmlFor of the field to the select identifier.
+   */
+  id: PropTypes.string.isRequired,
```

Comment:
This is a little tricky, should the ID be required when used outside of a field? There won't be an htmlFor to be bound to

Reply:
Updated to be optional within the select, but required in the field.